crates/bevy_sprite/src/render/mod.rs

```diff
@@ -186,6 +187,20 @@ pub struct ExtractedSprite {
     pub flip_y: bool,
 }
 
+#[derive(Clone, Copy)]
+pub struct Transform2d {
+    pub translation: Vec2,
+    pub scale: Vec2,
+    pub rotation: Quat,
+}
+
+impl Transform2d {
```

Comment:
There are a *large* number of convenience methods and conversions I would like for this, but IMO we should keep the scope of this PR small.